editor/src/clj/editor/properties_view.clj

```diff
@@ -1019,89 +210,533 @@
+        (.addEventFilter slider MouseEvent/MOUSE_PRESSED on-mouse-pressed)
+        (.addEventHandler slider MouseEvent/MOUSE_RELEASED on-mouse-released)
+        (.addListener (.valueProperty slider) value-changed-listener)
+        #(do
+           (.removeEventFilter slider MouseEvent/MOUSE_PRESSED on-mouse-pressed)
+           (.removeEventHandler slider MouseEvent/MOUSE_RELEASED on-mouse-released)
+           (.removeListener (.valueProperty slider) value-changed-listener)
+           (vreset! value-changed-fn nil))))
+    fx.lifecycle/callback))
+
+(defmethod make-control-view :slider [property _context localization-state]
+  (let [old-num (coalesced-property->old-num property)
+        is-float (math/float32? old-num)
+        {:keys [min max precision]
+         :or {min 0.0 max 1.0}} (:edit-type property)
+        value (properties/unify-values (properties/values property))
+        read-only (properties/read-only? property)]
+    {:fx/type fxui/grid
+     :spacing :small
+     :column-constraints [{:fx/type fx.column-constraints/lifecycle :percent-width 20}
+                          {:fx/type fx.column-constraints/lifecycle :percent-width 80}]
+     :children
+     [(-> {:fx/type fxui/value-field
+           :grid-pane/column 0
+           :pref-column-count (if precision (count (str precision)) 5)
+           :to-string field-expression/format-number
+           :to-value field-expression/to-double
+           :value value
+           :on-value-changed #(set-values! property (repeat (cond-> % is-float float)))
+           :editable (not read-only)}
+          (resolve-validation property localization-state))
+      {:fx/type fx.slider/lifecycle
+       :grid-pane/column 1
+       :focus-traversable false
+       :min min
+       :max max
+       :disable read-only
+       :value (or value max)
+       prop-slider-on-value-changed
+       (fn make-on-value-changed []
+         (let [op-seq (gensym)]
+           (fn on-value-changed [new-value]
+             (let [new-value (if precision
+                               (math/round-with-precision new-value precision)
+                               new-value)]
+               (set-values! property (repeat (if is-float (float new-value) new-value)) op-seq)))))}]}))
+
+(defmethod make-control-view :multi-line-text [property _context localization-state]
+  (-> {:fx/type fxui/value-area
+       :min-height 68.0
+       :pref-height 68.0
+       :on-value-changed #(set-values! property (repeat %))
+       :editable (not (properties/read-only? property))}
+      (resolve-value property)
+      (resolve-validation property localization-state)))
+
+(defmethod make-control-view :default [property _context localization-state]
+  (-> {:fx/type fxui/value-field
+       :to-string str
+       :editable false}
+      (resolve-value property)
+      (resolve-validation property localization-state)))
+
+;; TODO: profile
```

Comment:
I did some testing and used the scale tool with ~1000 objects selected. It was slightly slower in this case, but the old implementation would not refresh the properties view at all during drag. I think it's fine. It would be nice to not have to evaluate the `:_properties` outputs at all during the drag operation, though. A later improvement for sure!